src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1263,6 +1264,80 @@ public ResponseEntity<String> getMapLines(
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getMapLines(studyUuid, nodeUuid, substationsIds, inUpstreamBuiltParentNode));
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/dynamic-simulation/mappings")
+    @Operation(summary = "Get all mapping of dynamic simulation on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "All mappings of dynamic simulation"),
+        @ApiResponse(responseCode = "204", description = "No dynamic simulation mappings"),
+        @ApiResponse(responseCode = "404", description = "The dynamic simulation mappings has not been found")})
+    public ResponseEntity<List<MappingInfos>> getDynamicSimulationMappings(@Parameter(description = "study UUID") @PathVariable("studyUuid") UUID studyUuid,
+                                                                               @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        List<MappingInfos> mappings = studyService.getDynamicSimulationMappings(nodeUuid);
+        return mappings != null ? ResponseEntity.ok().body(mappings) :
+                ResponseEntity.noContent().build();
+    }
+
+    @PostMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/dynamic-simulation/run")
+    @Operation(summary = "run dynamic simulation on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The dynamic simulation has started")})
+    public ResponseEntity<UUID> runDynamicSimulation(@Parameter(description = "studyUuid") @PathVariable("studyUuid") UUID studyUuid,
+                                                     @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid,
+                                                     @Parameter(description = "mappingName") @RequestParam("mappingName") String mappingName,
+                                                     @RequestBody(required = false) String parameters) {
```

Comment:
Could the DTO too?

Reply:
I don't think it is necessary to return a (big) Dto because this end point is used only to trigger an action. The result uuid even does not used at the moment but I keep it for the same convention as other computations and may be useful in the future.

In my opinion, only GET endpoints need return a Dto

Reply:
I mean for the parameters: "String parameters"

Reply:
Using Dto DynamicSimulationParametersInfos instead of String...
Done!